rustls/src/lib.rs

```diff
@@ -338,6 +338,10 @@ pub mod internal {
     pub mod msgs {
         pub use crate::msgs::*;
     }
+    /// Low-level TLS message decryption functions.
+    pub mod cipher {
+        pub use crate::cipher::*;
```

Comment:
Please explicitly list the exported items here, rather than using a wildcart export.

Reply:
Done in commit 7e94bf6